### ISSUE TITLE
Clarify the quantization bits in the pretrained models?

### ISSUE THREAD

**Issue by kentaroy47** [pytorch/vision#1854]
Thanks for the great work, and quantized pretrained models had been added in torchvision 0.5.
https://github.com/pytorch/vision/releases

>Quantized models
torchvision now provides quantized models for ResNet, ResNext, MobileNetV2, GoogleNet, InceptionV3 and ShuffleNetV2, as well as reference scripts for quantizing your own model in references/classification/train_quantization.py (https://github.com/pytorch/vision/blob/master/references/classification/train_quantization.py). 

However, I was confused what is the quantized bits this models are in.
Is it in FP16 or INT8? I think this should be clarified to lessen confusion.

**Comment by fmassa** [comment]
Hi,

Thanks for the issue.

We have added INT8 quantized models to the release, following the (experimental) quantization support added in PyTorch 1.3, see https://github.com/pytorch/pytorch/releases/tag/v1.3.0

I'm not sure if it's ok to update the release notes after they have been published, but we can definitely make a more explicit mention in the README in https://github.com/pytorch/vision/tree/master/references/classification , and PRs improving it are welcome.

**Comment by kentaroy47** [comment]
@fmassa 
Thank you for the quick reply.
Yes I will make a PR to document this in the weekends

**Comment by kentaroy47** [comment]
raised PR 👀

**Comment by fmassa** [comment]
Thanks for the PR @kentaroy47 !
